### PR TITLE
NEWS.md: interim update

### PR DESCRIPTION
**Pull request by johanmalm**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -28,6 +29,53 @@ The format is based on [Keep a Changelog]
...
+- Add `policy` option to `AutoPlace` action. PR #1784
+- Add window type filter to If-actions. Written-by: @xi PR #1731
+- Add screen magnifier which can be controlled with the `ZoomIn`, `ZoomOut` and
+  `ToggleMagnify` actions. Written-by: @spl237 PR #1774
+
+### Fixed
+
+- Prevent child views from opening outside of usable area. PR #1878
+- Fix IME popups issues (flicker when popup surface is initially mapped
+  and bug which incorrectly shows multiple popups). PR #1872
```
Comment (Consolatis) [inline NEWS.md:55]: the "and bug" part seems wrong.

File: NEWS.md [hunk 2]
```diff
@@ -28,6 +29,53 @@ The format is based on [Keep a Changelog]
...
+- Rate-limit cursor-driven resize events based on monitor's refresh rate. This
+  fixes the lag when resizing windows of some apps on XWayland, for example
+  Chromium and Steam. PR #1861
+- Session-lock: fix flashing & update cursor shape. PR #1858
+- Remove tearing-controller listeners on destroy. PR #1853
+- Fix action parsing bug. PR #1838
```
Comment (Consolatis) [inline NEWS.md:61]: Suggest something like "Handle invalid `ForEach` and `If` action configs" instead.

File: NEWS.md [hunk 3]
```diff
@@ -28,6 +29,53 @@ The format is based on [Keep a Changelog]
 | 2021-04-15 | [0.2.0]       | 0.13.0          | 5011          |
 | 2021-03-05 | [0.1.0]       | 0.12.0          | 4627          |
 
+## [unreleased]
+
+### Added
+
+- Add `onRelease` option to `<keybind>` in support of binding `Super_L` to a
+  menu. Written-by: @spl237 PR #1888
+- Add support for `security-context-v1`. Written-by: @nesteroff PR #1817
```
Comment (Consolatis) [inline NEWS.md:38]: Suggest "initial support" as user configurable blocklists are still missing.

File: NEWS.md (hunk 1 above)
Comment (johanmalm) [reply NEWS.md]: Done

File: NEWS.md (hunk 2 above)
Comment (johanmalm) [reply NEWS.md]: Done

File: NEWS.md (hunk 3 above)
Comment (johanmalm) [reply NEWS.md]: Done

**Pull request merged**